examples/ibm-catalog/standard-solution/variables.tf

```diff
@@ -0,0 +1,150 @@
+variable "slz_workspace_id" {
+  description = "IBM Cloud Schematics workspace ID of an existing Secure Landing Zone with VSIs deployment.If you do not yet have an existing deployment, click [here](https://cloud.ibm.com/catalog/content/slz-vpc-with-vsis-a87ed9a5-d130-47a3-980b-5ceb1d4f9280-global#create) to create one. Please note: a specific  configuration is needed for the deployment. You may find it [here](https://github.com/terraform-ibm-modules/terraform-ibm-powervs-infrastructure/tree/main/examples/ibm-catalog/standard-solution/slz_json_configs_for_powervs/vpc_landscape_config.json). Copy and paste that configuration into the `override_json_string` deployment value."
+  type        = string
+  default     = null
+}
+
+variable "pvs_zone" {
+  description = "IBM Cloud PVS Zone. Valid values: sao01,osa21,tor01,us-south,dal12,us-east,tok04,lon04,lon06,eu-de-1,eu-de-2,syd04,syd05"
+  type        = string
```

Comment:
Could we add a validation block here to check the values

Reply:
@surajsbharadwaj: let's do it in next iteration
@vburckhardt: is that ok to do it in next iteration?

Reply:
Yes - just track the action at https://github.com/terraform-ibm-modules/terraform-ibm-issue-tracker